src/index.html

```diff
@@ -24,16 +24,23 @@
         <div class="container">
           <form>
             <input class="search-bar" type="text" name="city-name" placeholder="Enter the city" />
-            <button class="submit-btn" type="submit"></button>
+            <button class="submit-btn" type="submit">></button>
```

Comment:
Usuń dodatkowy >

Reply:
według mnie to nie dodatkowy znak a juz tekst przycisku aby imitował strzałke wiec spoko